### DISCUSSION TITLE
Welcome to fellows Discussions!

### DISCUSSION THREAD

**Discussion by rexyagami** [incubateind/fellows#2] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by evanessa234** [comment]
Hi! this is Heena Singh here, I saw this as one of the newly added project for Github Externship. This project has the tech stack which I am experienced with NodeJS, Express, mongodb, reactjs. I am so excited to see this project appear onto the list for GE. Can someone direct me where can I learn more about the project and the community?

Thanks in advance :)

**Comment by Priyaraj17** [comment]
Hey, this is Priyaraj, I am interested in this project, I believe this project is at its earliest stage right now, I am looking forward to work on it. It also suits my tech stack as I am experienced with the **MERN** stack. I have worked on a lot of projects related to it. I am looking forward to being involved with the project for a long time and learn from others. I would love some more inputs related to the project.

Thank you
